cmd/create/config.go

```diff
@@ -20,9 +20,9 @@ func NewCmdCreateCfg() *cobra.Command {
 		},
 	}
 
-	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "Add plugins")
-	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "Name of config file")
-	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "cluster info config")
+	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "add plugins")
+	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "cluster name")
```

Comment:
is this --name? Shall we support -a?

Reply:
`-n` can be supported. But `-n` may have other meanings，such as `namespace`.  So I didn't add it.